front/pages/api/w/[wId]/labs/transcripts/[tId].ts

```diff
@@ -33,18 +33,17 @@ async function handler(
   >,
   auth: Authenticator
 ): Promise<void> {
-  const userId = auth.user()?.id;
-
-  if (!userId) {
+  if (!auth.isUser()) {
```

Comment:
Are we sure that `auth.isUser()` checks for a valid plan of the workspace?

Reply:
It does not but the wrapper does